src/Propel/Generator/Builder/Om/ObjectBuilder.php

```
@@ -1015,6 +1015,10 @@ protected function addArrayAccessorBody(&$script, Column $column)
      */
     protected function addBooleanAccessor(&$script, Column $column)
     {
+        $name = $column->getCamelCaseName();
+        if (in_array($name, ClassTools::getPropelReservedMethods())) {
+            return; // Skip boolean accessors for reserved names
+        }
```

Instead of silently skipping should we print a error instead?